src/helpers/sortMethods.js

```diff
@@ -0,0 +1,13 @@
+export const sortByLatest = (list, bool) => {
```

Comment:
Sugestion: Indexing and query can also be used, if fits use-case and requirements.

Reply:
yeah will do that.